spec/systems/users_spec.rb

```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Signup', type: :system do
+  context 'validates screen' do
+    it 'displays signup screen should show correct field' do
+      visit signup_path
+
+      expect(page).to have_content('Signup')
+
+      within 'form' do
+        expect(page).to have_field('user[username]')
+        expect(page).to have_field('user[password]')
+        expect(page).to have_field('user[password_confirmation]')
+      end
+    end
+
+    it 'displays signup screen show correct label' do
```

Comment:
is it the same with `displays signup screen should show correct field` above?

Reply:
My bad. This test case I want to test if i18n work properly for label. Should be `have_content` not `have_field`

Reply:
Fixed in https://github.com/gutakk/ruby-google-scraper/pull/4/commits/99878e0d33ea843326f0431fd14f5d2c27587d57